app/src/main/AndroidManifest.xml

```diff
@@ -36,6 +36,9 @@
       android:label="@string/activity_social_feed"
       android:screenOrientation="portrait" />
 
+    <activity android:name=".speaker.SpeakerListActivity"
```

Comment:
For the other activities we are using fully qualified names, let's stick with that